astro/src/content/docs/customize/look-and-feel/simple-theme-editor.mdx

```diff
@@ -228,3 +229,13 @@ All the hosted pages are ordered into various categories, and you can click on a
 Additionally, you can click on the `Browser preview` button to open the currently selected page in a new browser tab. The page in the new tab will still have the style updates applied as long as the theme editor is active. This allows you to open several pages and once and see your style changes applied in real time.
 
 <img src="/img/docs/customize/look-and-feel/simple-theme-editor/preview.png" alt="Theme preview panel" />
+
+## Troubleshooting
+
+<ThemeTroubleshooting />
```

Comment:
Changed a bit because I think it is possible you could hose up a simple theme so much you could have difficulty logging in (if you changed everything to black or set all the button messages to the empty string or something).